### PR TITLE
MSI-2592: Alternative for IsProductSalableInterface when working with a list of items.

### PR DESCRIPTION
**Pull request by nmalevanec**
### Fixed Issues (if relevant)

1. https://github.com/magento/inventory/issues/2592 Alternative for IsProductSalableInterface when working with a list of items
2. https://github.com/magento/inventory/issues/2719 Introduce a Bulk version of IsProductSalableForRequestedQtyInterface API 

### Manual testing scenarios (*)

1. Create orders with all available product types and verify no errors occurred.
2. Export and import all available product types and verify no errors occurred.
3. Verify API calls, including GraphQl and verify no errors occurred during requests.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: InventorySalesApi/etc/webapi.xml [hunk 1]
```diff
@@ -19,12 +19,30 @@
...
             <resource ref="Magento_InventorySalesApi::stock"/>
         </resources>
     </route>
+    <route url="/V1/inventory/are-product-salable-for-requested-qty/" method="GET">
+        <service class="Magento\InventorySalesApi\Api\AreProductsSalableForRequestedQtyInterface" method="execute"/>
+        <resources>
+            <resource ref="Magento_InventorySalesApi::stock"/>
+        </resources>
+    </route>
+    <route url="/V1/inventory/are-product-salable-for-requested-qty/" method="GET">
```
Comment (sidolov) [inline InventorySalesApi/etc/webapi.xml]: Looks like we have two identical endpoints

**Review by sidolov — CHANGES_REQUESTED** [review]

**Comment by maghamed** [comment]
API changes and deprecation look good to me (/)

**Review by sidolov — APPROVED** [review]

**Comment by sidolov** [comment]
Approval ticket: https://jira.corp.magento.com/browse/MC-33023

**Comment by sidolov** [comment]
- SVC failures are expected, changes affects not released interfaces and acceptable for now.

**Pull request merged**